### ISSUE TITLE
No .el files with package headers in `/' when running `eldev -d version`

### ISSUE THREAD

**Issue by Silex** [doublep/eldev#21]
Hello,

Can you help me debug this?

```
silex@silex-laptop:~$ docker run -it --rm silex/emacs:dev bash
root@c2b0bbe27ff2:/# eldev -dvtQ --version
Bootstrapping Eldev for Emacs 27.1 from MELPA Stable...

Started up on Sun Sep 6 08:18:17 2020
Running on GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30)
 of 2020-08-20
Project directory: `/'
No file `/root/.eldev/config', not applying user-specific configuration
No file `Eldev', project building uses only defaults
No file `Eldev-local', not customizing build

Debugger entered--Lisp error: (error "No .el files with package headers in `/'")
  signal(error ("No .el files with package headers in `/'"))
  error("No .el files with package headers in `%s'" "/")
  package-dir-info()
  eldev--package-dir-info()
  eldev-package-descriptor()
  eldev-version()
  apply(eldev-version nil)
  eldev-parse-options(("-dvtQ" "--version") nil t)
  eldev-cli(("-dvtQ" "--version"))
  (kill-emacs (eldev-cli (append (cdr (member "--" command-line-args)) nil)))
  eval((kill-emacs (eldev-cli (append (cdr (member "--" command-line-args)) nil))) t)
  command-line-1(("--execute" "(let ((eldev--emacs-version (format \"%s.%s\" emacs-..." "--execute" "(kill-emacs (eldev-cli (append (cdr (member \"--\" c..." "--" "-dvtQ" "--version"))
  command-line()
  normal-top-level()

  backtrace()
  kill-emacs(-1)
  #f(compiled-function (&rest args) "Enter debugger.  \\<debugger-mode-map>`\\[debugger-continue]' returns from the debugger.\nArguments are mainly for use when this is called from the internals\nof the evaluator.\n\nYou may call with no args, or you may pass nil as the first arg and\nany other args you like.  In that case, the list of args after the\nfirst will be printed into the backtrace buffer.\n\nIf `inhibit-redisplay' is non-nil when this function is called,\nthe debugger will not be entered." (interactive nil) #<bytecode 0x1559b5890515>)(error (error "No .el files with package headers in `/'"))
  apply(#f(compiled-function (&rest args) "Enter debugger.  \\<debugger-mode-map>`\\[debugger-continue]' returns from the debugger.\nArguments are mainly for use when this is called from the internals\nof the evaluator.\n\nYou may call with no args, or you may pass nil as the first arg and\nany other args you like.  In that case, the list of args after the\nfirst will be printed into the backtrace buffer.\n\nIf `inhibit-redisplay' is non-nil when this function is called,\nthe debugger will not be entered." (interactive nil) #<bytecode 0x1559b5890515>) (error (error "No .el files with package headers in `/'")))
  #f(compiled-function (original &rest arguments) #<bytecode 0x1559b5a0df81>)(#f(compiled-function (&rest args) "Enter debugger.  \\<debugger-mode-map>`\\[debugger-continue]' returns from the debugger.\nArguments are mainly for use when this is called from the internals\nof the evaluator.\n\nYou may call with no args, or you may pass nil as the first arg and\nany other args you like.  In that case, the list of args after the\nfirst will be printed into the backtrace buffer.\n\nIf `inhibit-redisplay' is non-nil when this function is called,\nthe debugger will not be entered." (interactive nil) #<bytecode 0x1559b5890515>) error (error "No .el files with package headers in `/'"))
  apply(#f(compiled-function (original &rest arguments) #<bytecode 0x1559b5a0df81>) #f(compiled-function (&rest args) "Enter debugger.  \\<debugger-mode-map>`\\[debugger-continue]' returns from the debugger.\nArguments are mainly for use when this is called from the internals\nof the evaluator.\n\nYou may call with no args, or you may pass nil as the first arg and\nany other args you like.  In that case, the list of args after the\nfirst will be printed into the backtrace buffer.\n\nIf `inhibit-redisplay' is non-nil when this function is called,\nthe debugger will not be entered." (interactive nil) #<bytecode 0x1559b5890515>) (error (error "No .el files with package headers in `/'")))
  debug(error (error "No .el files with package headers in `/'"))
  signal(error ("No .el files with package headers in `/'"))
  error("No .el files with package headers in `%s'" "/")
  package-dir-info()
  eldev--package-dir-info()
  eldev-package-descriptor()
  eldev-version()
  apply(eldev-version nil)
  eldev-parse-options(("-dvtQ" "--version") nil t)
  eldev-cli(("-dvtQ" "--version"))
  (kill-emacs (eldev-cli (append (cdr (member "--" command-line-args)) nil)))
  eval((kill-emacs (eldev-cli (append (cdr (member "--" command-line-args)) nil))) t)
  command-line-1(("--execute" "(let ((eldev--emacs-version (format \"%s.%s\" emacs-..." "--execute" "(kill-emacs (eldev-cli (append (cdr (member \"--\" c..." "--" "-dvtQ" "--version"))
  command-line()
  normal-top-level()
```

**Comment by tsuu32** [comment]
`eldev version` works well on random directory (e.g. home) but `eldev -d version` on random directory shows same error.

**Comment by Silex** [comment]
Ah, interesting that `--version` makes an error while `version` does not:

```
philippe@pv-desktop:~$ docker run -it --rm silex/emacs:dev bash -c 'eldev --version; echo $?'
Bootstrapping Eldev for Emacs 27.1 from MELPA Stable...
eldev 0.7.1
1

philippe@pv-desktop:~$ docker run -it --rm silex/emacs:dev bash -c 'eldev version; echo $?'
Bootstrapping Eldev for Emacs 27.1 from MELPA Stable...
eldev 0.7.1
0
```

**Comment by doublep** [comment]
Reproduced, will release a fix soon.

**Comment by doublep** [comment]
Fixed both reported problems in 0.7.2.
